ftcnn/modeling/utils.py

```diff
@@ -2152,3 +2167,55 @@ def plot_paired_images(paired_images, nrows=1, ncols=2, figsize=(15, 15)):
 
     plt.tight_layout()
     plt.show()
+
+def clean_shapeFile(shapeFile):
```

Comment:
@Oasiss999 This function should return a gdf that retains the original data, or combines rows that have intersecting geometry. Can this be modified to make this possible? This way, the user can still have the metadata of the shapefile.